docs/running-on-kubernetes.md

```diff
@@ -602,4 +608,20 @@ specific to Spark on Kubernetes.
    <code>spark.kubernetes.executor.secrets.spark-secret=/etc/secrets</code>.
   </td>
 </tr>
+<tr>
```

Comment:
Please add a link to the docs here. https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables

Reply:
sure.

Reply:
@foxish I added the link the properties bellow like the other type of secrets is it ok now?

Reply:
LGTM, thanks @skonto; will merge once tests pass.